--- a/src/comgr-unbundle-command.cpp	2026-07-23 21:39:58.408357565 +0200
+++ b/src/comgr-unbundle-command.cpp	2026-07-23 21:49:31.676675098 +0200
@@ -15,6 +15,7 @@
 #include <comgr-unbundle-command.h>
 
 #include <clang/Driver/OffloadBundler.h>
+#include <llvm/Object/OffloadBundle.h>
 #include <llvm/BinaryFormat/Magic.h>
 
 namespace COMGR {
@@ -143,7 +144,7 @@
 
   MemoryBuffer &InputBuffer = **MaybeInputBuffer;
 
-  using Header = CompressedOffloadBundle::CompressedBundleHeader;
+  using Header = llvm::object::CompressedOffloadBundle::CompressedBundleHeader;
   Expected<Header> MaybeHeader = Header::tryParse(InputBuffer.getBuffer());
   if (!MaybeHeader)
     return MaybeHeader.takeError();
